learning-pathways/biont-python.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: learning-pathway
+
+title: "From Zero To Hero with Python"
+description: |
+  Data analysis and programming are high-demand fields, making them particularly attractive to job seekers and employees across various industries and academic institutions. Are you eager to acquire the foundational skills of coding using the Python programming language? The [BioNT consortium](https://biont-training.eu/index.html) is offering a training curriculum for job seekers, SME employers, and everyone who would like to gain new skills in the field of life sciences. 
+  Follow our BioNT self-paced learning Python introduction course using Galaxy as a programming environment, and the learning materials of the [Carpentries](https://carpentries.org/). 
+
+tags: [python, programming, introduction, beginner]
+type: use
+
+
+cover-image: assets/images/BioNT_Logo.png
+cover-image-alt: BioNT logo a blue had with the text BioNT
+
+editorial_board:
+  - teresa-m
+  - hvelab
+  - silviadg87
+  - lisanna
+
+funding:
+  - biont
+
+pathway:
+  - section: "Set up your compute environment"
+    description: |
+      Get started by setting up your IPython notebook within Galaxy and installing the needed dependencies. You can choose any Galaxy instance you like for this training, we tested the training on US, EU, and AU Galaxy instance.
+    tutorials:
+      - name: galaxy-intro-jupyter
+        topic: galaxy-interface
+
+  - section: "Python course form Zero to Hero"
+    description: |
+      Below you will find the three half-days of the BioNT Python introduction course on the Galaxy Training Academy 2025 program. Please follow the self-paced learning materials.
+    tutorials:
+      - type: custom
+        name: "[Day1 - First steps](http://swcarpentry.github.io/python-novice-gapminder/01-run-quit.html)"
+        description: |
+          [Lecture Hands-on](http://swcarpentry.github.io/python-novice-gapminder/01-run-quit.html); [<i class="fas fa-video" aria-hidden="true"></i> Lecture Video](https://lhumos.org/player/5/0/661d1e94e4b08465bc30cbc9/661d1e9be4b08465bc30cbcd)
+      - type: custom
+        name: "[Day2 - Tabular data manipulation and visualisation](http://swcarpentry.github.io/python-novice-gapminder/07-reading-tabular.html)"
+        description: |
+          [Lecture Hands-on](http://swcarpentry.github.io/python-novice-gapminder/07-reading-tabular.html); [<i class="fas fa-video" aria-hidden="true"></i> Lecture Video](https://lhumos.org/player/5/1/661d1e94e4b08465bc30cbc9/661d1eb2e4b08465bc30cbd6)
+      - type: custom
+        name: "[Day3 - Advanced programming ](http://swcarpentry.github.io/python-novice-gapminder/13-conditionals.html)"
+        description: |
+          [Lecture Hands-on](http://swcarpentry.github.io/python-novice-gapminder/13-conditionals.html); [<i class="fas fa-video" aria-hidden="true"></i> Lecture Video](https://lhumos.org/player/5/2/661d1e94e4b08465bc30cbc9/661d1ec9e4b08465bc30cbdd)
```

Comment:
"Lecure Hands-on" sounds like a confusing description, is it a lecture (just listen/read) or hands on? if both, then just add an & orso?

Reply:
Sorry @shiltemann: I overlooked your comment. It is the Carpentries materials, so similar to our hands-on tutorials. Do we have a definition of what a lecture is? I guess only Hands-on would be the best description. I will update it soon.

Reply:
I updated it already and called it "Carpentries Tutorial" and "Tutorial Video" now, but feel free to adjust if you want something different :)